sale_order_invoicing_finished_task/views/product_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record
+        id="product_template_form_view_invoice_policy_inherit_sale_project"
+        model="ir.ui.view"
+    >
+        <field name="model">product.template</field>
+        <field
+            name="inherit_id"
+            ref="sale_project.product_template_form_view_invoice_policy_inherit_sale_project"
+        />
+        <field name="arch" type="xml">
+            <field name="project_id" position="before">
+                <field
+                    name="invoicing_finished_task"
+                    invisible="(type, '!=', 'service') or (service_tracking not in ['task_global_project', 'task_in_project'])"
```

Comment:
This expression is wrong, but I think it does not address my comment. It is more related to where the field is added to the view, not the invisible attribute.

```suggestion
                    invisible="(type != 'service') or (service_tracking not in ['task_global_project', 'task_in_project'])"
```

Reply:
Changes are done, take a look when you can.